gui/exported.go

```diff
@@ -36,8 +36,12 @@ func (app *App) GoJoinNetworkByToken(token string) (any, error) {
 
 // App.GoGetKnownNetworks returns all known network configs (node, server)
 func (app *App) GoGetKnownNetworks() ([]Network, error) {
-	configs := make([]Network, 0, 5)
+	// read fresh config from disk
+	if err := config.RefreshConfigs(); err != nil {
+		return nil, err
+	}
 
+	configs := make([]Network, 0, 5)
```

Comment:
is there a reason to use make vice
config := []Network{} or
var config []Network

Reply:
so creation and memory allocation done in one step

Reply:
in most cases (one network only) will be allocating more memory than required. In other cases(joining more than 5 networks), the append will result in reallocation.  Don't see the point, but not a huge deal.
For reference: google go style guide

Size hints and preallocation are important steps when combined with empirical analysis of the code and its integrations, to create performance-sensitive and resource-efficient code.

Most code does not need a size hint or preallocation, and can allow the runtime to grow the slice or map as necessary. It is acceptable to preallocate when the final size is known (e.g. when converting between a map and a slice) but this is not a readability requirement, and may not be worth the clutter in small cases.

Warning: Preallocating more memory than you need can waste memory in the fleet or even harm performance. When in doubt, see GoTip #3: Benchmarking Go Code and default to a zero initialization or a composite literal declaration.

Reply:
right. got it